watertap/property_models/tests/test_seawater_prop_pack.py

```diff
@@ -332,3 +334,45 @@ def configure(self):
             ("flow_mass_phase_comp", ("Liq", "TDS")): 1239.69,
             ("enth_mass_phase", "Liq"): 3.8562e5,
         }
+
+
+@pytest.mark.unit
+def test_list_properties(capsys):
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.props = props.SeawaterParameterBlock()
+
+    # clear any existing captured output, call list_properties, then capture its output
+    capsys.readouterr()
+    m.fs.props.list_properties()
+    captured = capsys.readouterr().out
+    expected_output = """
```

Comment:
tets for string matches are not a good idea imho. at a minimum use a regex to look for some things least likely to change.